### PR TITLE
add background color in AppCard

### PR DESCRIPTION
**Pull request by AreyouHappy**
### add background color
Easily distinguish your app cards.

```
{
  "apps": [{
    "serverURL": "",
    "appId": "",
    "masterKey": "",
    "appName": "",
    "iconName": "",
    "backgroundColor":"#000000"
  }],
  "iconsFolder": "icons"
}
```

### CODE REVIEW

**Comment by dplewis** [comment]
Can you provide a screenshot?

**Comment by AreyouHappy** [comment]
Here!

**Comment by dplewis** [comment]
Thanks for getting started on this featured request! https://github.com/parse-community/parse-dashboard/issues/681

Can you add the background color to the data browser?

untitled

**Comment by AreyouHappy** [comment]
Hmm ... 
I do not recommend it.

**Comment by natanrolnik** [comment]
Thanks for your PR, @AreyouHappy .
If no value is configured, what value is used? Can you make a condition that, if `backgroundColor` isn't set, or if it's set to an empty string, it is not used?

**Comment by flovilmart** [comment]
In my side I’m not sure I’m convinced by the feature.

**Comment by dplewis** [comment]
@flovilmart It helps people like me with multiple servers. I can't count how many times I've changed production thinking it was on local. Adding colors would help.

**Comment by AreyouHappy** [comment]
If the backgorundcolor is not entered, it is replaced with the default. ( It is CSS style)
The colors at Data brower don't look good and aren't in now.
Please let me know if you have a problem with that commit!
Thank you for your feedback.

**Comment by dplewis** [comment]
It’s not that it doesn’t look good it should be an indicator that a different server is being used. 

Changing the color of the app card is the same as looking at the URL.

I would like to see something more semi-themed to avoid confusion.

**Comment by AreyouHappy** [comment]
Hmm .. I don't have good sense of design.
Do you want the Data browser part to change color, too?
[1] Core part
[2] Classes part (Browser)
I'm concerned about how to make the difference between [1] and [2] colors.
Do you have any good ideas?

**Comment by dplewis** [comment]
@AreyouHappy You could have a `primaryBackgroundColor` and `secondaryBackgroundColor`  with defaults for both.

**Comment by AreyouHappy** [comment]
Would you try it?

**Comment by dplewis** [comment]
@AreyouHappy Thanks for the update. I'll have some time today to try it out!

File: Parse-Dashboard/parse-dashboard-config.json [hunk 1]
```diff
@@ -4,7 +4,8 @@
     "appId": "",
     "masterKey": "",
     "appName": "",
-    "iconName": ""
+    "iconName": "",
+    "backgroundColor":"#000000"
```
Comment (dplewis) [inline Parse-Dashboard/parse-dashboard-config.json]: Can you change background color here to primary and secondary defaults?

**Comment by dplewis** [comment]
@AreyouHappy Just tried it out and it works great. Just a small change. 

Defaults are working without adding these fields.

**Comment by AreyouHappy** [comment]
```
{
  "apps": [{
    "serverURL": "",
    "appId": "",
    "masterKey": "",
    "appName": "",
    "iconName": "",
    "primaryBackgroundColor": "",
    "secondaryBackgroundColor": ""
  }],
  "iconsFolder": "icons"
}
```
I changed it as follows.

**Comment by dplewis** [comment]
@natanrolnik How does this look?

**Comment by natanrolnik** [comment]
@dplewis I understand the need, and in my opinion, it's good: we still have the default version, and whoever wants to customize, it's his/her responsibility to do so. Where are the primary and secondary colors applied, in the end?

**Comment by dplewis** [comment]
@natanrolnik In the app cards and databrowser. The above attached images.

**Comment by natanrolnik** [comment]
Thanks @AreyouHappy and @dplewis.
Release 1.2.0 is already available :simple_smile:

**Pull request merged**